portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -57,12 +57,45 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8148 | Test Summary: Individuals usage update in real time"
-	@ignore = "true"
 	@priority = "5"
 	test IndividualsUpdatesInRealTime {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		ACUtils.launchDXP();
 
-		// TODO LRAC-8148 Usage#IndividualsUpdatesInRealTime pending implementation
+		JSONUser.addUser(
```

Comment:
@AfonsoHSantos, you need to login with this user you created

Reply:
I think you'd better create a `Site Name` page and connect the AC to that page. Because to make sure that only the **TEST TEST** login when you go back to DXP is not affecting these numbers

Reply:
fixed